docker-compose.yml

```diff
@@ -111,4 +112,4 @@ services:
         /bin/sh -c "
         rm -f /tmp/.X99-lock &&
         Xvfb :99 -screen 0 1024x768x16"
-        
\ No newline at end of file
+        
```

Comment:
Could this line be removed since it has spaces at the start and there are two empty lines in a row.